CHANGES.md

```diff
@@ -1,4 +1,189 @@
+## stable-2.3.0
+
+### Announcing Linkerd 2.3 🎈
```

Comment:
For previous stable releases, we've only included this subheading in the release notes on github, and not in CHANGES.md. I'd recommend removing it from here.